css-four-column-layout/styles.css

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Barlow', sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+  background-color: #f7f8fa;
```

Comment:
You should apply the `background-color` to the `body` element, which will make the background color fill the whole viewport.